test/integration/integration_admin_test.cc

```diff
@@ -129,6 +129,21 @@ TEST_P(IntegrationAdminTest, Admin) {
   initialize();
 
   BufferingStreamDecoderPtr response;
+  // With https://github.com/envoyproxy/envoy/pull/23921 ClusterInfo::trafficStats is lazy init.
+  // We need to trigger creation of ClusterInfo::trafficStats() by calling the * operator.
+  absl::Notification n;
+  test_server_->server().dispatcher().post([&]() {
+    (void)*test_server_->server()
+        .clusterManager()
+        .clusters()
+        .getCluster("cluster_0")
+        ->get()
+        .info()
+        ->trafficStats();
+    n.Notify();
+  });
+  n.WaitForNotification();
```

Comment:
now that I see this pattern a second time, can you factor it into a common include file for integration tests? Maybe wherever the type for test_server_ is declared?